app/models/profiles_search.rb

```diff
@@ -32,6 +32,7 @@ def detailed_search_result
     return Profile.none if @query.values.all? &:blank?
     result = Profile
       .where('city ILIKE :city', city: "%#{@query[:city]}%")
+      .where('country ILIKE :country', country: "%#{@query[:country]}%")
```

Comment:
Great. You got the tests running!!! Because you added the country field to the seeds.

But this shows another problem with our tests ( we are not great at testing I guess we have to write some more). Because in reality we have another problem which appears now because just a few profiles have a country field yet. :disappointed:  
If we use this line of code ( line 35 ) our `detailed search` page just shows profiles who have a country field.

So I guess you still have to add an `if` clause.
```
   if @query[:country].present?
      result = result
        .where('country ILIKE :country', country: "%#{@query[:country]}%")
     end
```

Does that make sense?